From 95a0d368e284c5d3956b9a0f9c965575576b9617 Mon Sep 17 00:00:00 2001
From: slitaz-cn <teasiu@163.com>
Date: Tue, 30 Mar 2021 12:02:03 +0800
Subject: [PATCH 09/13] set some default packages

Signed-off-by: slitaz-cn <teasiu@163.com>
---
 include/target.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/target.mk b/include/target.mk
index 2a81a7fc80..6e58c5d2fc 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -13,11 +13,11 @@ __target_inc=1
 DEVICE_TYPE?=router
 
 # Default packages - the really basic set
-DEFAULT_PACKAGES:=base-files libc libgcc busybox dropbear mtd uci bash opkg netifd fstools uclient-fetch logd urandom-seed urngd
+DEFAULT_PACKAGES:=base-files libc libgcc busybox dropbear mtd uci bash luci wget curl luci-compat block-mount fdisk lsblk bind-client opkg netifd fstools uclient-fetch logd urandom-seed urngd
 # For nas targets
 DEFAULT_PACKAGES.nas:=block-mount fdisk lsblk mdadm
 # For router targets
-DEFAULT_PACKAGES.router:=dnsmasq iptables ip6tables ppp ppp-mod-pppoe firewall odhcpd-ipv6only odhcp6c kmod-ipt-offload
+DEFAULT_PACKAGES.router:=dnsmasq-full iptables ip6tables ppp ppp-mod-pppoe firewall odhcpd-ipv6only odhcp6c kmod-ipt-offload
 DEFAULT_PACKAGES.bootloader:=
 
 ifneq ($(DUMP),)
-- 
2.25.1

